internal/config/config.go

```diff
@@ -0,0 +1,166 @@
+package config
```

Comment:
please use envconfig

Reply:
is it some external package? or do you mean to rename the package name?

Reply:
it's a name of the pacakge for simplifying work with configs